solr/core/src/test/org/apache/solr/update/TestNestedUpdateProcessor.java

```diff
@@ -107,8 +107,8 @@ public void testDeeplyNestedURPGrandChild() throws Exception {
     };
     indexSampleData(jDoc);
 
-    assertJQ(req("q", IndexSchema.NEST_PATH_FIELD_NAME + ":*/grandChild#*",
-        "fl","*",
+    assertJQ(req("q", IndexSchema.NEST_PATH_FIELD_NAME + ":*/grandChild",
+        "fl","*, _nest_path_",
```

Comment:
This change (and others here) to a test of an URP that isn't modified in this issue underscores my point made previously having the test for that URP be more of a unit test of what the URP produces (test the SolrInputDocument), and _not_ executing queries.  I'm not saying it was wrong to make this change in this issue but just want you to reflect on the ramifications of these choices.